src/shared/lib/user-benefits-api.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+
+// list of benefits we're looking for
+const BenefitEnum = z.enum([
+	'adFree',
+	'allowRejectAll',
+	'hideSupportMessaging',
+]);
```

Comment:
Can we get a comment detailing exactly where we're getitng this list from, e.g. is there something in DCR/user-benefits-api repo we can point to?